rust/agama-server/src/software/web.rs

```diff
@@ -312,6 +317,55 @@ async fn register(
     }
 }
 
+/// returns list of registered addons
+///
+/// * `state`: service state.
+#[utoipa::path(
+    get,
+    path = "/registration/addons/registered",
+    context_path = "/api/software",
+    responses(
+        (status = 200, description = "List of registered addons, used by Agama CLI", body = Vec<AddonParams>),
```

Comment:
This description goes to the OpenAPI spec. So I would omit the CLI part.
```suggestion
        (status = 200, description = "List of registered addons", body = Vec<AddonParams>),
```